### ISSUE TITLE
Support for ES6 import format

### ISSUE THREAD

**Issue by SujitSingh** [karandpr/cordova-plugin-android-rootbeer#1]
Hi, 

Can we please implement support for ES6 import format for this plugin? 

For reference you can reffer to this [issue](https://github.com/WuglyakBolgoink/cordova-plugin-iroot/issues/1). 
This will help us to import this plugin without compilation errors in Ionic/Angular(2 and above) without [workaround](https://github.com/WuglyakBolgoink/cordova-plugin-iroot/issues/1#issuecomment-357939841).

**Comment by karandpr** [comment]
Sure. I will look into it in a week or so. 👍

**Comment by karandpr** [comment]
@SujitSingh . Plugin is using module.exports .... 
Are you facing any issues importing ?

**Comment by SujitSingh** [comment]
@karandpr yes it generates error saying "Cannot find module "cordova/exec". Import statement is - 
```
import cdvRootBeer from 'cordova-plugin-android-rootbeer/www/plugin';
```

I am currently working on [Ionic framework](https://ionicframework.com). The thing is, Ionic doesn't include `cordova` by default, which is required in this plugin. Thus the TypeScript compilation fails because it couldn't find `cordova` instance.
